langs/bash/Dockerfile

```diff
@@ -1,15 +1,18 @@
 FROM alpine:3.20 AS builder
 
-RUN apk add --no-cache build-base curl patch
+RUN apk add --no-cache build-base curl
 
-RUN curl https://ftp.gnu.org/gnu/bash/bash-5.2.tar.gz | tar xz && cd bash-5.2 \
- && curl https://ftp.gnu.org/gnu/bash/bash-5.2-patches/bash52-0[01-32] | patch -p0
+RUN curl https://ftp.gnu.org/gnu/bash/bash-5.2.tar.gz \
+  | tar xz --strip-components 1
```

Comment:
If we're going to unpack into root maybe we ought to use a workdir for safety, there's a good chance tarbal directories could collide with system directories

Reply:
> If we're going to unpack into root maybe we ought to use a workdir for safety, there's a good chance tarbal directories could collide with system directories

Good idea.